scripts/sct_run_batch.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+##############################################################################
+#
+# Wrapper to processing scripts, which loops across subjects. Data should be
+# organized according to the BIDS structure:
+# https://github.com/sct-pipeline/spine_generic#file-structure
+#
+# ----------------------------------------------------------------------------
+# Copyright (c) 2020 Polytechnique Montreal <www.neuro.polymtl.ca>
+# Authors: Julien Cohen-Adad, Chris Hammill
+#
+# About the license: see the file LICENSE.TXT
+##############################################################################
+
+from __future__ import absolute_import
+
+import os
+import sys
+import argparse
+import numpy as np
+import multiprocessing
+import subprocess
+import re
+import time
+
+import spinalcordtoolbox.image as msct_image
+from spinalcordtoolbox.image import Image, concat_data
+from spinalcordtoolbox.utils import Metavar, SmartFormatter
+
+parser = argparse.ArgumentParser(
+  description = 'Wrapper to processing scripts, which loops across subjects. Data should be '
+                'organized according to the BIDS structure: '
+                'https://github.com/sct-pipeline/spine_generic#file-structure.\n'
+                'The processing script (task) should accept a subject directory as its only argument. '
+                'Additional information is passed via environment variables \n'
+                'PATH_SEGMANUAL (--path-segmanual)\n'
+                'PATH_DATA (--data-path)\n'
+                'PATH_RESULTS (--out-path)\n'
+                'PATH_LOG (--out-path with log appended)\n'
+                'PATH_QC (--out-path with QC appended)\n'
+                'ITK_GLOBAL_DEFAULT_NUMBER_OF_THREADS (--itk-threads)\n',
+  formatter_class = SmartFormatter,
+  prog=os.path.basename(__file__).strip('.py'))
+
+parser.add_argument("--jobs", type = int, default = 1,
+                    help = 'The number of jobs to run in parallel. '
+                           'either an integer greater than or equal to one '
+                           'specifying the number of cores, 0 or a negative integer '
+                           'specifying number of cores minus that number. For example '
+                           '--jobs -1 indicates run ncores - 1 jobs in parallel.')
+parser.add_argument("--data-path", help = 'Path containing subject directories in the BIDS format')
```

Comment:
to be consistent with "--path-segmanual", and also to follow [SCT's variable naming conventions](https://github.com/neuropoly/spinalcordtoolbox/wiki/variables#variables)
```suggestion
parser.add_argument("--path-data", help = 'Path containing subject directories in the BIDS format')
```

Reply:
Make sense, I will fix all the parser arguments to match the conventions in the document above.

Reply:
Should all follow convention now